kolibri/core/templates/kolibri/unsupported_browser.html

```diff
@@ -32,6 +31,7 @@
       padding: 32px;
     }
   </style>
+  {% frontend_base_head_markup %}
```

Comment:
I had forgotten it should be here, and this is a good catch - but also making me think that maybe using the frontend_base_head_markup might not be the right way to do this, and we should just make a separate and specific shortcut icon template tag instead.

My reasoning being that anything can be injected into the tag, and people may not be testing for non-compatible browser compatibility, so erroring javascript could end up in here by accident and make the unsupported browser page unreliable.